src/Mvc/src/Microsoft.AspNetCore.Mvc.Core/BindingSourceKey.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace Microsoft.AspNetCore.Mvc
```

Comment:
Could this be in the `ModelBinding` namespace? I imagine it's not super common for users to need this, no?

Reply:
Leaving as is because `[BindProperty]` and `[ModelBinder]` are in the top-level Mvc namespace, meaning an extra `using` would be needed